Corona-Warn-App/src/main/res/layout/contact_diary_location_bottom_sheet_fragment.xml

```diff
@@ -39,12 +53,14 @@
             app:counterMaxLength="250"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/contact_diary_location_bottom_sheet_close_button">
+            app:layout_constraintTop_toBottomOf="@id/contact_diary_location_bottom_sheet_close_button">
```

Comment:
why remove the + ? i know it works but we've always left it there before

Reply:
+ is wrong because it does not add an id. I saw it, so I removed it.

Reply:
isn't the + just: 'create the id if it does not exist yet'? If the id is already defined it should just act as a normal reference, as would be the case here.

I'm just suprised since i have not seen any AS complaints about this and i think it even generates constraints with the @+ notation by default.